src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -46,22 +49,36 @@
                 xhr.send(payload || '');
             },
 
-            hasClass = function(el, klass) {
-                return el.className && el.className.match(new RegExp('\\b' + klass + '\\b'));
+            hasClass = function(el, cssClass) {
+                if (el.classList) {
```

Comment:
checking for classLiast support could be done before defining the function instead of doing it on each call of the function. It would be much more efficient